docs/containers-transports.5.md

```diff
@@ -43,7 +43,10 @@ The containers-registries.conf(5) further allows for configuring various setting
 Note that a _docker-reference_ has the following format: `name[:tag|@digest]`.
 While the docker transport does not support both a tag and a digest at the same time some formats like containers-storage do.
 Digests can also be used in an image destination as long as the manifest matches the provided digest.
+
+The docker transport supports pushing images without a tag or digest to a registry when the image name is suffixed with **@@unknown-digest@@**. The `name`**@@unknown-digest** reference format cannot be used with a reference that has a tag or digest.
```

Comment:
Expressing this as `name`**@@unknown-digest** expresses the difference between literals and variables in HTML output, but not in man pages; in man pages (with the go-md2man we use), both render as bold.

Italic (`_name_`) renders in man pages as underline, that makes the distinction work in both output formats.

Also, please update the `name[:tag|@digest]` above to express the difference between literals and variables (and [ | ] meta-syntax).